app/views/api/v2/analytics/user_analytics/show.html.erb

```diff
@@ -13,178 +13,180 @@
+  </div>
 
-<!-- monthly hypertension control view -->
-<div class="card">
-  <h3>Hypertension control</h3>
-  <p class="desc">
-    How many patients with hypertension had BP below 140/90 at their last follow-up visit in the month?
-  </p>
-
-  <table class="progress-table">
-    <% @user_analytics.monthly_period_list.each do |month_date| %>
-      <tr>
-        <td class="row-label no-padding"><%= month_date.strftime("%B") %></td>
-        <td><span class="percent">
-              <%=
-                @user_analytics.display_percentage(@statistics.dig(:monthly,
-                                                                   :grouped_by_date,
-                                                                   :controlled_visits,
-                                                                   month_date),
-                                                   @statistics.dig(:monthly,
-                                                                   :grouped_by_date,
-                                                                   :total_visits,
-                                                                   month_date))
-              %>
-            </span></td>
-        <td>
-          <%= zero_if_unavailable(@statistics.dig(:monthly,
-                                                  :grouped_by_date,
-                                                  :controlled_visits,
-                                                  month_date)) %>
-          of
-          <%= zero_if_unavailable(@statistics.dig(:monthly,
-                                                  :grouped_by_date,
-                                                  :total_visits,
-                                                  month_date)) %>
-        </td>
-      </tr>
-    <% end %>
-  </table>
-</div>
+  <!-- monthly hypertension control view -->
+  <div class="card">
+    <h3>Hypertension control</h3>
+    <p class="desc">
+      How many patients with hypertension had BP below 140/90 at their last follow-up visit in the month?
+    </p>
+
+    <table class="progress-table">
+      <% @user_analytics.monthly_period_list.each do |month_date| %>
+        <tr>
+          <td class="row-label no-padding"><%= month_date.strftime("%B") %></td>
+          <td><span class="percent">
+                  <%=
+                    @user_analytics.display_percentage(@statistics.dig(:monthly,
+                                                                       :grouped_by_date,
+                                                                       :controlled_visits,
+                                                                       month_date),
+                                                       @statistics.dig(:monthly,
+                                                                       :grouped_by_date,
+                                                                       :follow_ups,
```

Comment:
This used to be called `total_visits` before, it's now called `follow_ups`. This is the only logical change to this file. The rest is just reformatting.